src/meinberlin/meinberlin/static/stylesheets/scss/components/_availability_indicator.scss

```diff
@@ -19,14 +19,14 @@ phase. If the feature is available, they have the status `is-available`.
 
     &:after {
         @include border-radius(50%);
+        @include rem(border, 0.15rem);
+        @include rem(bottom, 0.12rem);
+        @include rem(right, 0.05rem);
         content: "";
         position: absolute;
-        bottom: 0.12em;
-        right: 0.05em;
         display: block;
         width: 0;
         height: 0;
-        border: 0.15em solid;
```

Comment:
These values are not correct (smaller than before).

Also: The `solid` from `border` has gone missing.